# RViz Message Command sent to add_markers node
# command is either "add" or "delete"
string command
float32 pose_x
float32 pose_y
float32 orientation_z
float32 orientation_w
